packages/data-broker/src/__tests__/services/dhis/DhisService.fixtures.js

```diff
@@ -25,20 +25,42 @@ export const DATA_GROUPS = {
   }),
 };
 
+const DL_FACILITY_A = {
+  code: 'DL_FACILITY_A',
+  name: 'DL FACILITY A',
+  type: 'facility',
+  metadata: {},
+  isTrackedEntity: () => false,
```

Comment:
Mocking out `getParent` is still a bit more convenient since the real implementation relies on a whole lot of set up. But good point, I hadn't even thought of replacing these will real the `EntityType`. Will play with it and see how it looks 👍

Reply:
One thing I've noticed is we needlessly mock out the database. We could just insert the mock data and let it use the real db. But that's for a future PR. Happy with what you have if it works for now :)

Reply:
I tried with using the `EntityType`, but since we're not actually using a DB connection it could be a tad confusing (since the DatabaseType requires a model, which we'd have to stub out). Thinking of going with how it was originally if that's cool?

Reply:
Sure 👍